docs/api/dashboard/export-dashboard.asciidoc

```diff
@@ -9,7 +9,7 @@ experimental[] Export dashboards and corresponding saved objects.
 [[dashboard-api-export-request]]
 ==== Request
 
-`GET /api/kibana/dashboards/export`
+`GET <kibana host>:port/api/kibana/dashboards/export`
```

Comment:
Should port also be in angle brackets?
```suggestion
`GET <kibana host>:<port>/api/kibana/dashboards/export`
```

If so, the rest will need to be changed as well.